<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SANKAPPA" failures="0" tests="10" name="Class2.Case14" time="34.975" errors="1" timestamp="2023-11-11T15:40:40 IST" skipped="6">
  <testcase classname="Class2.Case14" name="orderconfirmvisible" time="0.000">
    <skipped/>
  </testcase> <!-- orderconfirmvisible -->
  <system-out/>
  <testcase classname="Class2.Case14" name="carddata" time="0.000">
    <skipped/>
  </testcase> <!-- carddata -->
  <system-out/>
  <testcase classname="Class2.Case14" name="details" time="0.000">
    <skipped/>
  </testcase> <!-- details -->
  <system-out/>
  <testcase classname="Class2.Case14" name="buttons" time="0.000">
    <skipped/>
  </testcase> <!-- buttons -->
  <system-out/>
  <testcase classname="Class2.Case14" name="address" time="0.000">
    <skipped/>
  </testcase> <!-- address -->
  <system-out/>
  <testcase classname="Class2.Case14" name="verifylastdetails" time="0.089">
    <skipped/>
  </testcase> <!-- verifylastdetails -->
  <system-out/>
  <testcase classname="Class2.Case14" name="testHomePageVisibility" time="0.145"/>
  <system-out/>
  <testcase classname="Class2.Case14" name="productselect" time="10.663"/>
  <system-out/>
  <testcase classname="Class2.Case14" name="productaddtocart" time="11.596"/>
  <system-out/>
  <testcase classname="Class2.Case14" name="details1" time="12.482">
    <error message="Cannot locate an element using xpath=//*[contains(text(),&#039;Enter Account Information&#039;)]
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;SANKAPPA&#039;, ip: &#039;192.168.1.4&#039;, os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;20.0.2&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Cannot locate an element using xpath=//*[contains(text(),'Enter Account Information')]
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SANKAPPA', ip: '192.168.1.4', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '20.0.2'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:327)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at Pom2.Pomscenario14.Signupverify(Pomscenario14.java:114)
at Pom2.Pomscenario14.signupIsVisible(Pomscenario14.java:118)
at Class2.Case14.details1(Case14.java:55)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- details1 -->
  <system-out/>
</testsuite> <!-- Class2.Case14 -->
